### ISSUE TITLE
Incorrect sizing on second trigger leads to clipping at right and bottom.

### ISSUE THREAD

**Issue by adrinux** [jackmoore/colorbox#315]
I'm having an issue with colorbox, it seems superficially similar to https://github.com/jackmoore/colorbox/issues/120 but much more reproducible and not fixable.

colorbox works fine when **first** triggered on a given page, but if its triggered for a **second** time the size of cboxContent is calculated smaller, resulting in either the image, or the faux border being clipped at right and bottom. Refreshing the page fixes the issue for one trigger, the second still shows the bug.

I've stripped my problem project right down to a minimal test case, with a chunky blue colorbox faux border to make the problem really obvious and set it up at:
http://cbox-test.perlucida.com

jquery-1.8.3.min.js
jQuery ColorBox v1.3.30
using a minimally changed colorbox.css from – iirc – example 2.

The following browsers show this bug:
Win7 IE9, Chrome 24.0.1312.56 m, OP 12.12
OSX Saf 6.02, Chrome  25.0.1364.45 beta

Win7 FF4, FF12 and OSX FF 19 beta all work normally and don't show this bug (so I presume FF will probably work in all versions 4+).

tl:dr = Firefox is OK, everything else shows the bug.

I can't see that I'm doing anything wrong. Am I?

I tried many things in the context of my project, including downgrading to jquery 1.7 and upgrading to 1.9. Still showed the problem.

If no one has any obvious suggestions I'll dig into the js and try and figure out what is going wrong tomorrow.
